package-lock.json

```diff
@@ -2817,9 +2817,9 @@
       }
     },
     "@uxpin/merge-cli": {
-      "version": "2.6.0",
-      "resolved": "https://registry.npmjs.org/@uxpin/merge-cli/-/merge-cli-2.6.0.tgz",
-      "integrity": "sha512-kjQz7gMZNLYv5wCUdeehEYw4oIFoqzoVtnA5rudBQ3w8RTMhCJI4u/4HFr8VnOHJtKTGc2UjjuHK8sEVV7CPZQ==",
+      "version": "2.6.2",
+      "resolved": "https://registry.uxpin.pl:4873/@uxpin%2fmerge-cli/-/merge-cli-2.6.2.tgz",
```

Comment:
I suppose we should use `npmjs.org`

Reply:
Ech… The global NPM configuration is messing up here